WordPress/WordPressUITests/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -38,8 +46,12 @@ class BlockEditorScreen: BaseScreen {
      - Parameter text: the test to enter into the title
      */
     func enterTextInTitle(text: String) -> BlockEditorScreen {
+        if titleViewBlog.waitForExistence(timeout: 5) {
+            titleViewBlog.doubleTap()
+            titleViewBlog.typeText(text)
+        } else {
         titleView.tap()
-        titleView.typeText(text)
+        titleView.typeText(text)}
```

Comment:
Closing braces should be placed on a [new line](https://github.com/wordpress-mobile/swift-style-guide#braces) in all cases

```suggestion
            titleView.typeText(text)
        }
```